packages/front-end/components/Radix/Badge.tsx

```diff
@@ -0,0 +1,13 @@
+import { Badge as RadixBadge } from "@radix-ui/themes";
+import { MarginProps } from "@radix-ui/themes/dist/cjs/props/margin.props";
+import { RadixColor } from "@/components/Radix/HelperText";
+
+type Props = {
+  children: string | JSX.Element;
```

Comment:
I've been trying to avoid accepting `JSX.Element` as a prop since I think it makes it too easy for engineers to deviate from the design system.

Reply:
If we're not accepting `JSX.Element` then I think that we should use another name for the parameter. `children` is the convention for child elements so it will confuse people if you can't use it this way. Also, react does some caching/mangling of `children`.

I had it initially as a string and switched to a full element to be able to add bold/styling to the content. I can also imagine use case where we might want to pass an icon etc.